<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="mainLayout">
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>463</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionGenerate"/>
    <addaction name="actionGeneratePack"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionHelpInfos"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGenerate"/>
   <addaction name="actionGeneratePack"/>
   <addaction name="actionOpenFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionGenerate">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/new/image/save</normaloff>:/new/image/save</iconset>
   </property>
   <property name="text">
    <string>Generate image</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/new/image/quit</normaloff>:/new/image/quit</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionGeneratePack">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/new/image/savePack</normaloff>:/new/image/savePack</iconset>
   </property>
   <property name="text">
    <string>Generate pack</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/new/image/open</normaloff>:/new/image/open</iconset>
   </property>
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionHelpInfos">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/new/image/info</normaloff>:/new/image/info</iconset>
   </property>
   <property name="text">
    <string>Infos</string>
   </property>
   <property name="toolTip">
    <string>Infos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
